src/main/java/seedu/address/commons/util/FileUtil.java

```diff
@@ -147,4 +170,23 @@ public static String joinPath(String ... tokens) {
     public static String getSeparator() {
         return System.getProperty("file.separator");
     }
+
+    /**
+     * Returns a {@code Byte} array given an {@code int} array. Wildcard bytes should be
+     * represented by a 32 bit integer with its MSB set.
+     *
+     * @param intArray the array to convert from.
+     * @return a {@code Byte[]} based on the values in {@code intArray}
+     */
+    public static Byte[] intArrayToByteArray(int[] intArray) {
+        Byte[] bytes = new Byte[intArray.length];
+        for (int i = 0; i < intArray.length; i++) {
+            if (intArray[i] < 0) {
+                bytes[i] = null;
+            } else {
+                bytes[i] = (Byte) ((byte) intArray[i]);
```

Comment:
Is this double casting necessary?

Reply:
`Byte` and `byte` is treated differently by java and apparently you need explicit casts here lmao